storage/object/reference-content/ssl-tls-certificate.mdx

```diff
@@ -0,0 +1,169 @@
+
+Note that in certain cases an intermediate certificate may not be necessary, if the root certificate chains directly to the primary/server certificate. The crucial thing is that the subject and issuers of each certificate form a coherent chain of validation. If a certificate is issued by an authority that is not present in the chain, an error will occur.
+
+<Message type="tip">
+
+You can use the [OpenSSL](https://www.openssl.org/) utility to convert certificates and keys from other formats to PEM, from the command line. Once installed, use a command like the following:
+
+```sh 
+openssl x509 -in cert.crt -out cert.pem
+```
+
+```sh 
+openssl x509 -in cert.der -out cert.pem
+```
+
+```sh 
+openssl x509 -in cert.cer -out cert.pem
+```
+
+When you have your key, your server certificate and your root certificate all in separate files, you can use the `cat` command to chain them together into one file, ready to be copied and pasted:
+
+```sh
+cat private_key.pem cert.pem root_cert.pem > cert_chain.pem
+```
+
+</Message>
+
+## Uploading your certificate
+
+When you [configure your customized domain](/storage/object/how-to/get-started-edge-services/#how-to-configure-a-custom-domain) with Edge Services for the first time, you are prompted to upload your certificate. You can do so in two ways:
+
+    - Select an existing certificate that you have stored in [Scaleway Secret Manager](/identity-and-access-management/secret-manager/quickstart/).
+    - Manually import a certificate into Scaleway Secret Manager, directly from the Edge Services **Configure domain** wizard (copy and paste the PEM formatted chain). Your certificate will be automatically stored in Secret Manager.
+
+## Keeping your certificate up to date
+
+SSL/TLS certificates all expire at some point. If your certificate expires before you upload a new one, you will see an error like this on your Edge Services dashboard:
+
+    <Lightbox src="scaleway-cert-expired.webp" alt="Your SSL/TLS certificate has expired. Update your certificate in Secret Manager. Your initial certificate will remain in use until the error is fixed." />
+
+You must renew your certificate or create a new one. A number of tools are available to ensure that certificates are automatically renewed before expiry, for example [Certbot for LetsEncrypt](https://eff-certbot.readthedocs.io/en/stable/using.html#renewing-certificates).
+
+When you have your up to date certificate, go to [Secret Manager](https://console.scaleway.com/secret-manager/secrets) in the console,  and access the secret that contains your certificate. [Create a new version](/identity-and-access-management/secret-manager/how-to/create-version/) of the secret, to hold the up to date certificate. Edge Services will automatically detect and use the most recent enabled version of the secret. You can nonetheless choose to disable or delete the old version(s) as you prefer.
```

Comment:
@RoRoJ, deleting unused version also allows reducing secret manager costs since you are paying per version stored.